spec/basket_spec.rb

```diff
@@ -6,7 +6,6 @@
   end
 
   before do
-    Basket.config
```

Comment:
I thought we needed this and I think we did some time ago but we don't now so i'm removing it.